syntax = "proto3";

package ProtoTransport;

message Stops {
    string name = 1;
    repeated string buses = 2;
}

message Buses {
    string name = 1;
    uint32 stops = 2;
    uint32 unique_stops = 3;
    uint32 road_route = 4;
    double linear_route = 5;
    repeated string route = 6;
    bool route_type = 7;
}

message Db {
    repeated Stops stops = 1;
    repeated Buses buses = 2;
}
